examples/transactions/examples/permit2_signature_transfer.rs

```diff
@@ -76,7 +76,7 @@ async fn main() -> Result<()> {
 
     // Create a provider with both signers pointing to anvil
     let rpc_url = anvil.endpoint_url();
-    let provider = ProviderBuilder::new().wallet(wallet).on_http(rpc_url);
+    let provider = ProviderBuilder::new().wallet(wallet).connect_http(rpc_url);
```

Comment:
Seeing a 

```
Sent approval: 0x93426765c406ba149df94b52df30b415bc812b7d384dbcaafb9a389a4eec3555
Error: server returned an error response: error code 3: execution reverted, data: "0x"
```

Does not appear to be a regression as it also appears on `main`

Still worth looking into

Reply:
Adding to the ignore list in `run.sh` for now

Reply:
added follow-up ticket for it here: https://github.com/alloy-rs/examples/issues/218